language-elixir_5bdc647204286356f0a56048.md

```diff
@@ -20,7 +20,7 @@ CI pipeline that you can use to get started quickly:
 
 Semaphore uses [kiex](https://github.com/taylor/kiex) to manage
 Elixir versions. Any version installable with kiex is supported on
-Semaphore. Version 1.7 is pre-installed. You may install new versions
+Semaphore. Version 1.7 and 1.8.1 is pre-installed. You may install new versions
```

Comment:
say 1.8 because minor version doesn't matter in this context and we say 1.7 just before